web/packages/teleport/src/services/user/types.ts

```diff
@@ -134,6 +134,10 @@ export interface User {
   allTraits?: AllUserTraits;
 }
 
+// Backend does not allow User field "traits" and "allTraits"
+// to be specified when creating or updating a user.
```

Comment:
```suggestion
// Backend does not allow User fields "traits" and "allTraits"
// both to be specified in the same request when creating or updating a user.
```
Mention both cannot be provided?